gqlalchemy/models.py

```diff
@@ -350,16 +350,22 @@ def field_in_superclass(field, constraint):
 
             return None
 
+        def get_base_labels() -> Set[str]:
+            base_labels = set()
+            nonlocal bases
+            for base in bases:
+                base_labels = base_labels.union(base.labels if hasattr(base, "labels") else set())
```

Comment:
feels like this union operation could be skipped if `hasattr` returns `False`

Reply:
Thank you for the suggestion. I skipped the union operation if `hasattr` returns `False`.